lib/modules/apostrophe-login/index.js

```diff
@@ -315,22 +368,52 @@ module.exports = {
             if (!user) {
               throw 'notfound';
             }
-            // Gets i18n'd in the template, also bc with what templates that tried to work
-            // before certain fixes would expect (this is why we still pass a string and not
-            // a flag, and why we call it `message`)
-            return self.sendPage(req, 'passwordReset', { reset: reset, email: email });
+            return self.sendPage(req, 'passwordReset', { reset: reset, email: email, message: getMessage(), errors: req.query.errors });
           }).catch(function(err) {
             self.apos.utils.error(err);
             return res.redirect('/login?message=' + req.__('That reset code was not found. It may have expired. Try resetting again.'));
           });
+
+          function getMessage() {
+            let message;
+            if (req.query.message) {
+              message = req.query.message;
+            }
+            if (req.query.errors && req.query.errors.length) {
+              message = req.query.errors.join(', ');
```

Comment:
This seems like it'll read weirdly. Why not write the messages as sentences and join with a space?

Reply:
That... would take less logic. Will do.